project_forecast_line/models/forecast_line.py

```diff
@@ -348,3 +348,26 @@ def unlink(self):
         # we routinely unlink forecast lines, let"s not fill the logs with this
         with mute_logger("odoo.models.unlink"):
             return super().unlink()
+
+    @api.model_create_multi
+    @api.returns("self", lambda value: value.id)
+    def create(self, vals_list):
+        records = super().create(vals_list)
+        employee_role_lines = records.filtered(
+            lambda r: r.res_model == "hr.employee.forecast.role"
+        )
+        if employee_role_lines:
+            # check for existing records which could have the new lines as
+            # employee_resource_forecast_line_id
+            other_lines = self.search(
+                [
+                    ("employee_resource_forecast_line_id", "=", False),
```

Comment:
if we're checking new `employee_resource_forecast_line_id` why we are filtering on empty?

Reply:
we want to find lines that could be linked to the created (we are in the create() method, we know which are the created lines) through the employee_resource_forecast_line_id, so we look for lines which don't already have a value in that field.